docker/test/integration/minifi/validators/SegfaultValidator.py

```diff
@@ -1,3 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
```

Comment:
I don't think the double new lines are necessary. 0 or 1 is enough IMO.

Reply:
flake8 insists on 2 blank lines between the `import`s and the start of the code, and also between top-level functions/classes: https://www.python.org/dev/peps/pep-0008/#blank-lines.  So 2 blank lines seemed consistent here, too (although flake8 does not enforce this).